### DISCUSSION TITLE
Docusaurus SEO: Discovered – currently not indexed

### DISCUSSION THREAD

**Discussion by arthurbackouche** [facebook/docusaurus#8740] [Q&A]
Hello,

In Google Search Console, most of the URLs related to my docusaurus website are: Discovered – currently not indexed

I've added the parameter "trailingSlash: true" however I'm still not sure that my config is correct.
![Screenshot 2023-03-07 at 3 55 17 pm](https://user-images.githubusercontent.com/12870604/223324321-0a239a58-1c03-4622-87dc-2700ed506ca8.png)

Could someone help me? Here's my repo: https://github.com/arthurbackouche/arthurbackouche

Thank you

**Comment by slorber** [comment]
Difficult to help without having the life production url of your site.

Google usually gives a list of reasons for a page to not being indexed here, so please make sure to give us all the details, including the screenshots of pages of those reasons.


BTW you are supposed to use a gitignore file: node_modules, build and .docusaurus are not supposed to be added to the git repo
